docs/java/features/resilience/caching.mdx

```diff
@@ -16,7 +16,7 @@ keywords:
 image:
 ---
 
-Caching may improve your applications performance and responsiveness to your consumers.
+Caching may improve your application`s performance and responsiveness to your consumers.
```

Comment:
This is actually not a typo. The *'s* is short for *is* similar to how *I'm* is short for *I am*. So *application's* would be short for *application is*. So *applications* is correct.

Reply:
I didn't get it, do you mean to say _application's_ can also be written as _applications_ here?

Reply:
Ah nevermind I missread it, sorry.

Reply:
No probs, I put the wrong character there too instead of an apostrophe.